ovs/lib/noderemoval.py

```diff
@@ -93,8 +93,8 @@ def remove_node(node_ip, silent=None):
             storage_router_to_remove_online = True
             for storage_router in storage_router_all:
                 try:
-                    client = SSHClient(storage_router, username='root')
-                except (UnableToConnectException, NotAuthenticatedException):
+                    client = SSHClient(storage_router, username='root', timeout=10)
+                except (UnableToConnectException, NotAuthenticatedException, TimeOutException):
```

Comment:
Are we sure at this point that the node that times out is the node that went down?
If not, we'll introduce completely different behaviour, because the node which isn't really down, but really slow in responding, will not be unconfigured properly because our logic assumes its down